src/components/common/ClickToCopy.js

```diff
@@ -37,7 +37,7 @@ const ClickToCopy = ({ className, children, copy, translate = 'default' }) => {
     const [show, setShow] = useState(false)
 
     const handleCopy = () => {
-        Mixpanel.track(`Copy ${copy} with click`)
+        Mixpanel.track(`Click to copy text`, {text: copy})
```

Comment:
@icerove is there a reason why we want to include `copy`? It's the users account ID

Reply:
I recommend that we remove `{name: copy}`